### PR TITLE
ACR 3.0 SDK (support ACR Build/Task GA Service)

### PR DESCRIPTION
**Pull request by northtyphoon**
## Description

Swagger: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/containerregistry/resource-manager
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/3617 

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by northtyphoon** [comment]
@djyou

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@northtyphoon do you need any other approvals or is this good to merge

File: src/SDKs/ContainerRegistry/Management.ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry.csproj
Comment (djyou) [inline src/SDKs/ContainerRegistry/Management.ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry.csproj]: NIT: do you want to mention this is a breaking change from the `2.8.0-preview` version since `build` is rebranded to `task`, although the major version bump already indicates that.

**Review by djyou — APPROVED** [review]
LGTM

File: src/SDKs/ContainerRegistry/Management.ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry.csproj
Comment (northtyphoon) [reply src/SDKs/ContainerRegistry/Management.ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry.csproj]: updated

**Comment by northtyphoon** [comment]
@dsgouda, it's ready to merge.

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Pull request merged**
